### PR TITLE
feat: codesniffer and rector fixes for Drupal 10

### PR DESCRIPTION
**Pull request by podarok**
Addresses: https://github.com/YCloudYUSA/yusaopeny/pull/8
Fixes: https://yusa.atlassian.net/browse/DS-64

### CODE REVIEW

File: modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module [hunk 1]
```diff
@@ -8,39 +8,42 @@
 use Drupal\node\Entity\Node;
 use Drupal\paragraphs\Entity\Paragraph;
 
- function openy_prgf_af4_demo_install() {
-     // Make sure that blocks are available.
+/**
+ *
```
Comment (podarok) [inline modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module]:
```suggestion
 * Install demo content while module enabling is happening.
```

File: src/Entity/ProgramSearchCheckLog.php [hunk 2]
```diff
@@ -84,7 +83,10 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     return $fields;
   }
 
-  static public function getTypeOptions() {
+  /**
+   *
```
Comment (podarok) [inline src/Entity/ProgramSearchCheckLog.php]:
```suggestion
   * List of options.
```

File: src/OpenyActivityFinderBackend.php [hunk 3]
```diff
@@ -2,18 +2,20 @@
 
 namespace Drupal\openy_activity_finder;
 
-use Drupal\Core\Config\Config;
 use Drupal\Core\Config\ConfigFactoryInterface;
 use Drupal\Core\StringTranslation\StringTranslationTrait;
 
+/**
+ *
```
Comment (podarok) [inline src/OpenyActivityFinderBackend.php]:
```suggestion
 * Implements interface.
```

File: src/OpenyActivityFinderBackendInterface.php [hunk 4]
```diff
@@ -2,6 +2,9 @@
 
 namespace Drupal\openy_activity_finder;
 
+/**
+ *
```
Comment (podarok) [inline src/OpenyActivityFinderBackendInterface.php]:
```suggestion
 * Interface for the backend. 
```

File: src/ProgramSearchLogViewsData.php [hunk 5]
```diff
@@ -4,7 +4,11 @@
 
 use Drupal\views\EntityViewsData;
 
+/**
+ *
```
Comment (svicervlad) [inline src/ProgramSearchLogViewsData.php]: Description required. Like a "Class to provide visitors counter for Program Search Log Views""

File: modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module [hunk 6]
```diff
@@ -69,9 +72,9 @@ use Drupal\paragraphs\Entity\Paragraph;
           if ($paragraph->bundle == 'activity_finder_v4') {
             $data['id'] = 'activity_finder_v4_block';
           }
-        //   elseif ($paragraph->bundle == 'activity_finder_search') {
-        //     $data['id'] = 'activity_finder_search_block';
-        //   }
+          // Elseif ($paragraph->bundle == 'activity_finder_search') {
```
Comment (svicervlad) [inline modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module]: Unused code, I think it is better to delete it

**Review by svicervlad — CHANGES_REQUESTED** [review]

File: modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module [hunk 7]
```diff
@@ -69,9 +72,9 @@ use Drupal\paragraphs\Entity\Paragraph;
...
+          //     $data['id'] = 'activity_finder_search_block';
+          //   }.
```
Comment (podarok) [inline modules/openy_prgf_activity_finder_4/openy_prgf_af4_demo/openy_prgf_af4_demo.module]:
```suggestion
```

File: src/ProgramSearchLogViewsData.php (hunk 5 above)
Comment (podarok) [inline src/ProgramSearchLogViewsData.php]:
```suggestion
 * Implements EntityViewsData.
```

**Pull request merged**